client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -58,46 +61,95 @@ public ReaderGroupConfigBuilder disableAutomaticCheckpoints() {
         * Add a stream and its associated start {@link StreamCut} and end {@link StreamCut} to be read by the
         * readers of a ReaderGroup.
         *
-        * @param scopedStreamName Scoped Name of the Stream.
-        * @param startStreamCut Start {@link StreamCut}
-        * @param endStreamCut End {@link StreamCut}
+        * @param scope Scope of the stream, if it is not specified then {@link ReaderGroup#getScope()} is used for the scope.
```

Comment:
Does "Not specified" mean null? if so just say "null"

Given that this method is overloaded, I don't think we need to document what if it is not specified, they can simply use the other method. This method might even disallow passing null.

Reply:
updated the comment.